scene/gui/option_button.cpp

```diff
@@ -131,6 +131,7 @@ void OptionButton::_notification(int p_what) {
 		case NOTIFICATION_TRANSLATION_CHANGED:
 		case NOTIFICATION_LAYOUT_DIRECTION_CHANGED: {
 			popup->set_layout_direction((Window::LayoutDirection)get_layout_direction());
+			popup->set_auto_translate(is_auto_translating());
 			[[fallthrough]];
 		}
```

Comment:
It's not necessary to do it in `NOTIFICATION_LAYOUT_DIRECTION_CHANGED`, probably should be another fall through case:

```suggestion
		case NOTIFICATION_TRANSLATION_CHANGED: {
			popup->set_auto_translate(is_auto_translating());
			[[fallthrough]];
		}
		case NOTIFICATION_LAYOUT_DIRECTION_CHANGED: {
			popup->set_layout_direction((Window::LayoutDirection)get_layout_direction());
			[[fallthrough]];
		}
```